.gitignore

```diff
@@ -18,12 +18,13 @@
 !/storage/.keep
 .byebug_history
 
-# Ignore master key for decrypting credentials and more.
-/config/master.key
+# Ignore credentials
+/config/credentials/production.key
+/config/credentials/staging.key
 
```

Comment:
```suggestion
/config/master.key
```
mind adding the old mechanism master key as well? This would alleviate if a dev tries to the non-env specific (still available) `rails credentials:edit` cmd and thus adds secrets with the key and possibly pushes to gh.

Reply:
✔️